fury/primitive.py

```diff
@@ -626,6 +626,55 @@ def prim_triangularprism():
     return vertices, triangles
 
 
+def prim_pentagonalprism():
+    """Return vertices and triangle for an pentagonal prism.
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our prism
+    triangles: ndarray
+        triangles that compose our prism
+
+    """
+    # Local variable to represent the square root of five rounded
+    # to 7 decimal places
+    five = float('{:.7f}'.format(math.sqrt(5)))
+    onec = (five - 1) / 4.0
+    twoc = (five + 1) / 4.0
+    sone = (math.sqrt(10 + (2 * five))) / 4.0
+    stwo = (math.sqrt(10 - (2 * five))) / 4.0
+
+    vertices = np.array([[-twoc, stwo, -1],
+                         [onec, sone, -1],
+                         [1, 0, -1],
+                         [onec, -sone, -1],
+                         [-twoc, -stwo, -1],
+                         [-twoc, stwo, 1],
+                         [onec, sone, 1],
+                         [1, 0, 1],
+                         [onec, -sone, 1],
+                         [-twoc, -stwo, 1]])
```

Comment:
are you sure that your coordinates are centered with (0,0,0)?

Reply:
Same comment as @SunTzunami, use more descriptive variable names.